### DISCUSSION TITLE
Notifying beginners about the Green square/ Contributing opportunities

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1782] [Ideas] [answered]
Ideas for notifying beginners when a new contribution opportunity is added

**Comment by kaiwalyakoparkar** [comment, marked as answer]
What if we make a portal where everyone can add their issues and everyone will get notified (preferably by email) that there is a contribution opportunity.
- As soon as the issue is closed, the issue will get auto removed from the portal
- This will keep the beginners updated with the contribution opportunities

**Reply by vinzvinci** [reply]
Maybe, it's better to use discord to notify them, we should have `first-timer` role, so we can specifically notify them.

**Reply by ghost** [reply]
Add me to the role then!

**Reply by kaiwalyakoparkar** [reply]
Haha me too :rofl:

**Reply by Vyvy-vi** [reply]
Moreover, one can pick up the github repos from `#good-first-issues` and _watch_ them
This way, they can get notified of contribution opportunities.
I'd also like to re-iterate, you both don't count as first-timers :P
